pallets/msa/src/lib.rs

```diff
@@ -419,8 +482,9 @@ pub mod pallet {
 			let (new_msa_id, new_public_key) =
 				Self::create_account(public_key, |_| -> DispatchResult { Ok(()) })?;
 
-			Self::deposit_event(Event::MsaCreated { msa_id: new_msa_id, key: new_public_key });
-
+			let event = Event::MsaCreated { msa_id: new_msa_id, key: new_public_key };
+			Self::index_event(event.clone());
```

Comment:
indexing an event

Reply:
This function continues to index events sent to it and also index the count of events it gets per block for offchain reference